packages/ui-components/src/lib/Library/index.js

```diff
@@ -80,10 +80,30 @@ const LibraryItemCards = styled.div`
 
 const LibraryComponent = (props) => {
   const [query, setQuery] = useState([]);
+  const [ciceroVersion, setCiceroVersion] = useState([]);
```

Comment:
When viewing the [demo](https://deploy-preview-187--ap-web-components.netlify.app/?path=/story/library--heterogeneous), it looks like the initial state of the `ciceroVersion` is an empty array so no templates are displayed. This should initialize with showing _all_ the template cards.

Reply:
That's a good idea. Will make the change.

Reply:
The initial state still starts as an empty state, but I am checking the length of the array in the new pr I pushed yesterday, so that all the UI templates get populated on initial load, when length is zero. They started getting filtered on selected version once array has length.

Reply:
Addressed commit message changes too.